CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## 0.10.3 (April 30, 2024)
+
+IMPROVEMENTS:
+* bump golang.org/x/net (https://github.com/hashicorp/vault-lambda-extension/pull/135)
+* bump go version, vault api, and vault sdk (https://github.com/hashicorp/vault-lambda-extension/pull/134)
```

Comment:
I wonder if we should state the versions for these updates?

Reply:
Sure thing! Done in https://github.com/hashicorp/vault-lambda-extension/pull/138/commits/00be257d4df870ab5f90b875405b8fbe442f5494